### PR TITLE
Reword Multiple Developers Branching Workflow

### PR DESCRIPTION
**Pull request by benhutton**

### CODE REVIEW

File: workflow/branching/multiple-developers.md [hunk 1]
```diff
@@ -1,34 +1,56 @@
...
-3. Other developers can submit PRs tagged as DO NOT MERGE if they need to deploy a review app and then close their PR when they are done.
+### Notes
+
+* As with pull requests against `master`, it is better to submit many small PRs than one large PR, if possible.
+
+* Never rebase directly off of `master`.
+
+* Never push directly to the owner's branch.
+
+* Use the "DO NOT MERGE" tage to submit a PR to deploy a review app, and then close the PR when finished.
```
Comment (joeosburn-desiringgod) [inline workflow/branching/multiple-developers.md]: @benhutton Typo on the word `tag` here.

**Comment by benhutton** [comment]
@joeosburn-desiringgod see new commit

**Comment by joeosburn-desiringgod** [comment]
@benhutton 👍 Do you want to squash before I merge these in? Or should I merge in and then squash myself?

**Comment by benhutton** [comment]
@joeosburn-desiringgod i think you should end up squashing into your original commit, so you might as well merge these then squash them all.

**Pull request merged**